ostree/ostree_transport_test.go

```diff
@@ -46,7 +46,7 @@ var imageNameTestcases = []struct{ input, normalized, branchName string }{
 	{"docker.io/library/busybox:latest", "docker.io/library/busybox:latest", "docker.io_2Flibrary_2Fbusybox_3Alatest"}, // A hierarchical name
 	{"127.0.0.1:5000/busybox:latest", "127.0.0.1:5000/busybox:latest", "127.0.0.1_3A5000_2Fbusybox_3Alatest"},          // Port usage
 	{"UPPERCASEISINVALID", "", ""},                                                                                     // Invalid input
-	{"busybox" + sha256digest, "", ""},                                                                                 // Digested references are not supported (parsed as invalid repository name)
+	{"busybox" + sha256digest, "busybox@sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef", "busybox_40sha256_3A0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"},
```

Comment:
I recommend placing this above the previous invalid input to keep the valid and invalids in their own blocks.

Reply:
amended and pushed